developer-docs-site/docs/concepts/base-gas.md

```diff
@@ -0,0 +1,240 @@
+
+### Vectors
+
+Byte-wise fees are similarly assessed on vectors, which consume $\sum_{i = 0}^{n - 1} e_i + b(n)$ bytes, where
+
+* $n$ is the number of elements in the vector,
+* $e_i$ is the size of element $i$,
+* and $b(n)$ is a "base size" which is a function of $n$.
+
+See [#4540] for more information on vector base size, which is typically just 1 byte in practice, such that a vector of 100 `u8` elements accounts for $100 + 1 = 101$ bytes.
+Hence per the item-wise read methodology described above, reading the last element of such a vector is treated as a 101-byte read.
+
+## Payload gas
+
+Payload gas is defined in [`transaction.rs`], which incorporates storage gas with several payload-associated parameters:
+
+| Parameter                       | Meaning                                                                                |
+|---------------------------------|----------------------------------------------------------------------------------------|
+| `min_transaction_gas_units`     | Minimum amount of gas for a transaction, charged at start of execution                 |
+| `large_transaction_cutoff`      | Size, in bytes, above which transactions will be charged an additional amount per byte |
+| `intrinsic_gas_per_byte`        | Units of gas charged per byte for payloads above `large_transaction_cutoff`            |
+| `maximum_number_of_gas_units`   | Upper limit on gas that a transaction can require                                      |
+| `min_price_per_gas_unit`        | Minimum gas price allowed on a transaction                                             |
+| `max_price_per_gas_unit`        | Maximum gas price allowed on a transaction                                             |
+| `max_transaction_size_in_bytes` | Maximum transaction payload size in bytes                                              |
+| `gas_unit_scaling_factor`       | Amount of gas units in one octal                                                       |
+
+<!--- Alphabetized reference links -->
+
+[#4540]:                           https://github.com/aptos-labs/aptos-core/pull/4540/files
+[`base_8192_exponential_curve()`]: ../../../aptos-move/framework/aptos-framework/build/AptosFramework/docs/storage_gas.md#0x1_storage_gas_base_8192_exponential_curve
+[`initialize()`]:                  ../../../aptos-move/framework/aptos-framework/build/AptosFramework/docs/storage_gas.md#0x1_storage_gas_initialize
+[`instr.rs`]:                      ../../../aptos-move/aptos-gas/src/instr.rs
+[`on_reconfig()`]:                 ../../../aptos-move/framework/aptos-framework/build/AptosFramework/docs/storage_gas.md#0x1_storage_gas_on_reconfig
+[`storage_gas.md`]:                ../../../aptos-move/framework/aptos-framework/build/AptosFramework/docs/storage_gas.md
```

Comment:
I think for the time being change this to the http url link to the file or leave it empty for now. After this PR is merged we can go back in and add the link target. I am guessing dangling url links do not break the docs build. Can you try https://github.com/aptos-labs/aptos-core/blob/e30f24f6c5a9b5bb0e1815284cd7278b28f83b1d/developer-docs-site/docs/concepts/base-gas.md and push the update, and let's see what happens.

Reply:
Just pushed the update, how's it looking?

Reply:
Looking great! Docs build passed. Next week I will give the doc one more pass in detail but they are good as they are. Thank you!

Reply:
Yeah @rajkaramchedu we may want to revise it a little bit before it goes live on the website

Reply:
Hi all, when will this file land in `main` for linking?
https://github.com/aptos-labs/aptos-core/blob/69d076700dbdbe5c49eb617937bd4999832397bd/aptos-move/framework/aptos-framework/build/AptosFramework/docs/storage_gas.md

I don't see it here now:
https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/framework/aptos-framework